R/predict.R

```diff
@@ -0,0 +1,25 @@
+#' @title
+#' Predict Individual Treatment Effect via Causal Rule Ensemble
+#'
+#' @description
+#' Predict Individual Treatment Effect via Causal Rule Ensemble algorithm.
+#'
+#' @param CRE A cre object from running the CRE function.
+#' @param X A covariate matrix (or data.frame)
+#'
+#' @return
+#' Array with the estimated Individual Treatment Effects
+#'
+#' @export
+#'
+predict.cre <- function(CRE, X) {
```

Comment:
In generic functions use `object` for consistency. This is section fails the tests.

Reply:
Oh good to know, thank you!